sirepo/job_supervisor.py

```diff
@@ -293,6 +302,7 @@ def _purge_sim(db_file):
             d.status = job.FREE_USER_PURGED
             cls.__db_write_file(d)
             jids_purged.append(db_file.purebasename)
+            cls._cache_purged_jid(d.computeJid)
```

Comment:
`jids_purged` is unnecessary, and use purebasename since that's what's used on [278](https://github.com/radiasoft/sirepo/pull/2485/files#diff-940c7c8cb02a032b1e9c4befe02e6b81R278).

I don't think the abstraction in this case helps. Rather:

```
cls._cache_purged_jid.add(db_file.purebasename)
```

Reply:
Can you give some context on why jids_purged is unnecessary? I have the same general question as [this comment](https://github.com/radiasoft/sirepo/pull/2485#discussion_r420877358) on when info is useful and when not. If I was debugging a bug possibly related to purging of sims I think this is info I'd want (when a purge for a particular jid happened in relation to everything else going on for that jid).

Reply:
We might want to log the time of the purge (or rather, the update to the db). I think that's useful. However, it's unlikely that we'll be able to pinpoint the time of the purge relative to the complaint that a wrong jid was purged. It's more like we'll get a complaint a week or month later, and we will look at the db first to see if it is purged, because that's easier to do than pinpointing when the purge happened in the logs.

Reply:
Make sense, thanks.